reactos/dll/win32/browseui/travellog.cpp

```diff
@@ -172,6 +172,7 @@ HRESULT STDMETHODCALLTYPE CTravelEntry::Update(IUnknown *punk, BOOL fIsLocalAnch
     GetToolTipText(punk, wch);
     TRACE("Updating entry with display name: %S\n", wch);
 
+    ZeroMemory(&windowData, sizeof(WINDOWDATA));
```

Comment:
Any particular reason to zero this? GetWindowData should fill it on success, there's nobody accessing it before or on failure.